guides/development-workflow.md

```diff
@@ -0,0 +1,284 @@
+
+**Convention:**
+```
+(feature): when we are adding something new
+(fix): when we are fixing something existing
+(refactor): when we are changing the implementation of something existing without changing its behaviour
+(chore): catch all for when none of the above apply and there is no immediate user need eg. gem/package updates
+```
+
+**Examples:**
+
+```
+# Poor
+Added repairs
+
+# Great
+(feature): Job postings are now displayed on the landing page
+
+* Getting the Job postings out of the external API proved difficult but I think the solution we have will allow us to easily upgrade to the new release they have planned: <link>
+```
+
+```
+# Poor
+Fixed the form
+
+# Great
+(fix): Food order form works with special characters
+
+* Previously the form failed to send if the starter value included any of the following characters: `!@£$%^&`
+* Users should be able to use these characters and after a short investigation I found this new bug in PHP: <link>
+* As a short term workaround until the bug is fixed I am stripping out any of these characters if they are present.
+```
+
+```
+# Poor
+Gem upgrade
+
+# Great
+(chore): Upgrade widget to v1.2.1
+
+* This widget now includes a patch that improves resilience: <link>
+* We are still unable to upgrade to v2.x as we need to do some migration work first
+```
+
+---
+
+## Working on tickets
+
+
+
+---
+
+## Code review
+
+
+## Code style
+We write our code in a consistent way to ensure it is well-structured and easy to follow for the rest of the team. Similar to the guidelines laid out in [Thoughtbot’s style guide](https://playbook.thoughtbot.com/#style-guide), approach this guidance as a way to code on present and future projects, not something to retroactively add to existing projects. When working on existing projects or joining a project, make sure to follow the code style that is already in place.
+
+While it can be tempting to write a new style guide, it's not a worthwhile investment of time. It's important that a consistent style is adopted, but mostly unimportant which specific style it is. So rather than having our own, we have chosen the following existing style guides for our work:
+
+* PHP - [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)
+* Ruby - [Ruby community style guide](https://github.com/bbatsov/ruby-style-guide)
+* Sass - [Sass Guidelines - Syntax and formatting](http://sass-guidelin.es/#syntax--formatting)
+* JavaScript - [Standard](http://standardjs.com/)
+
+---
+
+## Pull requests
+When we have finished a piece of work on a branch, we [make a pull request](https://help.github.com/articles/using-pull-requests/) using the project's GitHub/GitLab page.
+
+We prefix pull request titles with the Story ID to make it easy to find the story and the acceptance criteria that should be met.
```

Comment:
Why not put a link to the story in the body of the MR?

If you use `Resolves: https://...` then single-commit branches will automatically put the URL into the body of the MR.